NOLF2/ClientShellDLL/TO2/ScreenDisplay.cpp

```diff
@@ -199,12 +200,15 @@ LTBOOL CScreenDisplay::Build()
 	CLTGUIFrame *pBar = debug_new(CLTGUIFrame);
 	pBar->Create(0xBF000000,3,nHeight+1);
 	pBar->SetBasePos(pos);
-	pBar->SetScale(g_pInterfaceResMgr->GetXRatio());
+	pBar->ApplyPosition(g_pInterfaceResMgr->GetYRatio(), g_pInterfaceResMgr->Get4x3Offset());
+	pBar->SetScale(g_pInterfaceResMgr->GetYRatio());
```

Comment:
isn't this redundant... as `ApplyPosition` calls `CLTGUICtrl::SetScale`
and `CLTGUIFrame`'s `SetScale` function was renamed.

Reply:
I think this, and others like this should be addressed?